lib/media/track/localvideotrack.js

```diff
@@ -31,7 +31,7 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
    */
   constructor(mediaStreamTrack, options) {
     options = Object.assign({
-      workaroundSilentLocalVideo: guessBrowser() === 'safari' || isIOSChrome()
+      workaroundSilentLocalVideo: (guessBrowser() === 'safari' || isIOSChrome())
```

Comment:
Please address this.

Reply:
yeah, it would make sense to exclude tracks w/o deviceId - I will update. And since we will be using this logic at two places, I will move it to `utils!isUserMediaTrack`